code/controllers/subsystem/pullrequests.dm

```diff
@@ -0,0 +1,118 @@
+var/datum/subsystem/pullrequests/SSpullrequests
+
+/datum/subsystem/pullrequests
+	name = "Pull Requests"
+	priority = 0
+	wait = 120
+	can_fire = 1
+	var/nextPr
+	var/ignore_player_pref = 1
+
+/datum/subsystem/pullrequests/New()
+	nextPr = rand(17000, 17500)
+	NEW_SS_GLOBAL(SSpullrequests)
+
+/datum/subsystem/pullrequests/fire()
+	if(prob(10))
+		announcePr()
```

Comment:
Should make this a var too.

So i can vv wait to 5 and prob to 100 to spam them =P